### DISCUSSION TITLE
FooocusAPI on AWS Lambda

### DISCUSSION THREAD

**Discussion by Simone2498** [mrhan1993/FooocusWithFastAPI#23] [Q&A]
Hello everyone,
is it possible to deploy Fooocus API in an AWS lambda to avoid paying the host also when not used? Or would it be the loading time too long due to the stateless host and the download of the models? If it is possible, could you give me some indication on how to do it or a guide if there is one? :D

Thank you
Regards

**Comment by mrhan1993** [comment]
Sorry, but I didn't have much energy to study the use of these serverless platforms.
